### ISSUE TITLE
Exception while reading compilation error

### ISSUE THREAD

**Issue by peterwurzinger** [Taritsyn/DartSassHost#14]
Hi,

So I'm using DartSassHost in combination with the [Web Optimizer Stack](https://github.com/ligershark/WebOptimizer/) to minify and bundle some SASS files in a tryout ASP.NET Core MVC 8 application.

I encountered a strange behavior, because my application seems to emit an error message whenever the CSS bundle is requested. After digging through the code the core issue seems to be, that my stylesheets are malformed or something like that. But the problem is, that all I can see is a .NET internal exception message starting with:
```
An unhandled exception has occurred while executing the request.
     System.ArgumentException: The path is empty. (Parameter 'relativeTo')
         at System.IO.Path.GetRelativePath(String relativeTo, String path, StringComparison comparisonType)
         at DartSassHost.Helpers.PathHelpers.PrettifyPath(String currentDirectory, String path)
         ....
```

It seems to be, that an `IFileManager` returns an empty string for the current directory, effectively making its way by getting called
[here](https://github.com/Taritsyn/DartSassHost/blob/main/src/DartSassHost/JsonConverters/CompilationResultConverter.cs#L192), [passed from here](https://github.com/ligershark/WebOptimizer.Sass/blob/master/src/Compiler.cs#L85) (external to this repository) then effectively calling [this component here](https://github.com/ligershark/WebOptimizer.Sass/blob/d75eee0d35febcdf52471dd54170199d507e7a0e/src/FileProviderFileManager.cs#L27)

Long story short:
The [PathPrettifier](https://github.com/Taritsyn/DartSassHost/blob/15b2a914382f87bc0a41c4353ceca429b0ba2fe2/src/DartSassHost/Helpers/PathHelpers.cs#L23) tries to prettify a path with an empty string as current directory, which leads to `Path.GetRelativePath` throwing an exception.
It seems that the result of this call is subsequently used to create a compilation error message or something like that containing the information of the file location.

I suspect that there was already an attempt to fix this for [reading stack frames](https://github.com/Taritsyn/DartSassHost/blob/15b2a914382f87bc0a41c4353ceca429b0ba2fe2/src/DartSassHost/JsonConverters/CompilationResultConverter.cs#L554) and [reading warnings](https://github.com/Taritsyn/DartSassHost/blob/15b2a914382f87bc0a41c4353ceca429b0ba2fe2/src/DartSassHost/JsonConverters/CompilationResultConverter.cs#L456) from the compilation result, because my naive idea was a similar fix.

If you accept PRs, I'll happily supply a fix.

Cheers

**Comment by Taritsyn** [comment]
Hello, Peter!

Thanks for information! This error fixed in [version 1.0.14](https://github.com/Taritsyn/DartSassHost/releases/tag/v1.0.14).

**Comment by peterwurzinger** [comment]
Ooof, that was a quick one.
Thanks a ton, I can confirm this issue to be fiexed in 1.0.14

Btw. I was mixing up imports in a SASS file, and I can see the corresponding error message now :)
